source/common/upstream/upstream_impl.h

```diff
@@ -1331,7 +1358,7 @@ class PriorityStateManager : protected Logger::Loggable<Logger::Id::upstream> {
   // lb_endpoint health status is unhealthy, draining or timeout.
   void registerHostForPriority(
       const std::string& hostname, Network::Address::InstanceConstSharedPtr address,
-      const std::vector<Network::Address::InstanceConstSharedPtr>& address_list,
+      const HostDescription::AddressVector& address_list,
```

Comment:
Do we really want to have an alias for existing std::vector<Network::Address::InstanceConstSharedPtr>, which trigger these kind of changes?

Reply:
Yes I think it makes the code easier to read.